README.md

```diff
@@ -161,13 +285,13 @@ sender.SendSpan("getAllUsers", 1552949776000, 343, "localhost",
     },
     nil)
 ```
-**Note:** The tracing and span SDK APIs are designed to serve as low-level endpoints. For most use cases, we recommend using
-opentracing with the ```WavefrontTracer```.
-
-For more information on OpenTracing, please refer the OpenTracing project: https://github.com/opentracing/opentracing-go. To use OpenTracing with Wavefront, please refer to https://github.com/wavefrontHQ/wavefront-opentracing-sdk-go.
+***Note:*** The tracing and span SDK APIs are designed to serve as low-level endpoints. For most use cases, we recommend using
+the OpenTracing SDK with the `WavefrontTracer`.
+* See the [Go OpenTracing project](https://github.com/opentracing/opentracing-go) for details. 
+* To use OpenTracing with Wavefront, see the [Wavefront Go OpenTracing SDK](https://github.com/wavefrontHQ/wavefront-opentracing-sdk-go).
 
-#### Closing the Sender
-It is recommended to flush and close the sender before shutting down your application.
+## Close the Sender
+It is recommended to flush the buffer and close the sender before shutting down your application.
```

Comment:
VMware style tends to not use "It is recommended" unless there's a very compelling reason. How about: 

Before shutting down your application, flush the buffer and close the sender.